R/source.R

```diff
@@ -110,7 +110,7 @@ cpp_source <- function(file, code = NULL, env = parent.frame(), clean = TRUE, qu
   check_valid_attributes(all_decorations, file = orig_file_path)
 
   cli_suppress(
-    funs <- get_registered_functions(all_decorations, "cpp11::register")
+    funs <- get_registered_functions(all_decorations, "cpp11::register", quiet = quiet)
   )
```

Comment:
Was `cli_suppress()` already essentially forcing `quiet = TRUE`?

i.e. if you call `cpp11_source(quiet = FALSE)` right now, can you even force output to be shown? If not, I think we should think about removing `cli_suppress()` altogether and just hardcoding this to `quiet = TRUE`

Reply:
I guess now we can even remove `cli_suppress()` from the package.